src/convertToObject.js

```diff
@@ -6,7 +6,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrayWithStyles = complexStylesString.split(';')
```

Comment:
The variable `complexStylesString` is not defined within this function or passed as a parameter. This will cause a ReferenceError. You should use the `sourceString` parameter instead, as it seems to be intended for this purpose.